java/adapter/avro/src/main/java/org/apache/arrow/consumers/Consumer.java

```diff
@@ -22,9 +22,11 @@
 import org.apache.avro.io.Decoder;
 
 /**
- * An abstraction that is used to consume values from avro decoder.
+ * Interface that is used to consume values from avro decoder.
  */
 public interface Consumer {
 
   void consume(Decoder decoder) throws IOException;
+
+  void movePosition();
```

Comment:
Maybe something more literal like "addNull" this makes it clear what is being done.  

In either case please add documents for the intentended contract of each method.

Reply:
Sure, fixed.